doc/node.1

```diff
@@ -228,6 +228,9 @@ Among other uses, this can be used to enable FIPS-compliant crypto if Node.js is
 .It Fl -pending-deprecation
 Emit pending deprecation warnings.
 .
+.It Fl -policy-integrity=sri
```

Comment:
```suggestion
.It Fl -policy-integrity Ns = Ns Ar sri
```
(I think, based on the other examples in this file and the referenced https://man.openbsd.org/mdoc.7 at the top of this file.)